src/test/java/org/recap/controller/DataDumpRestControllerUT.java

```diff
@@ -124,7 +124,7 @@ public void exportIncrementalSCSBXmlFormatForHttp() throws Exception {
         String transmissionType = "1";
         String emailToAddress = "test@gmail.com";
```

Comment:
You have already declared the type of transmissionType, emailToAddress and response, why you need to define it every time, you can just say transmissionType = 1 ...